### PR TITLE
Rules now better ignore SCSS and Less variables.

### PR DESCRIPTION
**Pull request by evilebottnawi**
https://github.com/stylelint/stylelint/issues/1239

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 - Fixed: `declaration-block-no-ignored-properties` now detects use of `min-width` and `max-width` with inline, table-row, table-row-group, table-column and table-column-group elements.
 - Fixed: `declaration-block-no-ignored-properties` now detects use of `overflow`, `overflow-x` and `overflow-y` with inline elements.
 - Fixed: some rules now better handle case insensitive CSS identifiers.
+- Fixed: rules now better ignore SCSS and Less variables.
```
Comment (davidtheclark) [inline CHANGELOG.md]: Please name the rules.

**Comment by davidtheclark** [comment]
Great.

**Comment by umyar** [comment]
Hi, folks! What about situations, when i want do NOT ignore scss and less variables? For example I want rule `color-named: "never"` to show me an error in `@color: red;`? What should i do?

**Comment by evilebottnawi** [comment]
@umyar you need rule for this, you can open issue in `stylelint-scss`

**Comment by umyar** [comment]
> @umyar you need rule for this, you can open issue in `stylelint-scss`

Im using less. Is it issue for this repository?

**Comment by hudochenkov** [comment]
If you want to check this you'll have to write own plugin for that.

**Comment by umyar** [comment]
> If you want to check this you'll have to write own plugin for that.

I have a lot of rules for check in less variables, not only **color-named**. Is writing plugin a right way to resolve my issue? I think adding some flag for stylelint config (which allows linting less variables too) is more flexiable option, isnt it? 

Why scss and less variables are ignoring? It has some historical causes? 

P.S. Writing own plugin is hard to me, but not a problem at all. I just think, that this issue more global, than only my case. 

With love and peace, Umyar

**Comment by umyar** [comment]
Please, answer 😢 
What i have to do if i don't want to ignore less variables?

**Comment by evilebottnawi** [comment]
@umyar create plugin

**Comment by umyar1** [comment]
@evilebottnawi, thx for answer, bro! I will do it!

**Pull request merged**